### PR TITLE
Contact form UI improvments

### PR DESCRIPTION
**Pull request by sweatC**
Need to add styles for scroll and fix bug with text typing.
UPD: We have decided on call that it would be better to leave border bottom line for textarea and remove first and second lines.
image
image

### CODE REVIEW

**Comment by sweatC** [comment]
Staging: http://jt-pr278.surge.sh/
image
screenshot_2018-04-30-12-35-15

File: _assets/css/layout/_modal.scss [hunk 1]
```diff
@@ -116,7 +116,6 @@
     left: 0;
     right: 0;
     background: #424249;
-    height: 1px;
     top: 65px;
   }
   &:after {
```
Comment (andriyParashchuk) [inline _assets/css/layout/_modal.scss]: Need delete :before, :after selectors with property
pls chekc scss files in  _layot and _media

Comment (sweatC) [reply _assets/css/layout/_modal.scss]: @andriyParashchuk it will crash the layout

File: _assets/css/layout/_modal.scss [hunk 2]
```diff
@@ -140,9 +138,10 @@
   height: 65px;
   background: none;
   font-size: 16px;
-  line-height: 65px;
+  line-height: 20px;
```
Comment (andriyParashchuk) [inline _assets/css/layout/_modal.scss]: also need chache line-height in all scss files for different screen

File: _assets/css/layout/_modal.scss [hunk 3]
```diff
@@ -140,9 +138,10 @@
   height: 65px;
   background: none;
   font-size: 16px;
-  line-height: 65px;
+  line-height: 20px;
   color: $white;
   padding: 0;
+  margin-top: 15px;
```
Comment (andriyParashchuk) [inline _assets/css/layout/_modal.scss]: no need to indent all fields
psl add padding-top for only text-area

File: _assets/css/layout/_modal.scss (hunk 2 above)
Comment (andriyParashchuk) [reply _assets/css/layout/_modal.scss]: i think on desktop better line-height: 24px
pls change only text-area

Comment (sweatC) [reply _assets/css/layout/_modal.scss]: Accepted

File: _assets/css/layout/_modal.scss (hunk 3 above)
Comment (sweatC) [reply _assets/css/layout/_modal.scss]: Accepted

**Review by andriyParashchuk — APPROVED** [review]

**Review by obliviusm — APPROVED** [review]

**Pull request merged**